gsiopenssh/developer/index.adoc

```diff
@@ -14,7 +14,7 @@ This document provides information for GSI-OpenSSH developers.
 The changes to http://www.openssh.org/[OpenSSH] to add GSI support are
 limited, because we build on the existing GSSAPI support in OpenSSH for
 Kerberos. See the
-http://dev.globus.org/wiki/GSI-OpenSSH/Internals[GSI-OpenSSH Internals]
+https://web.archive.org/web/20130607001903/http://dev.globus.org:80/wiki/GSI-OpenSSH/Internals[GSI-OpenSSH Internals]
```

Comment:
Shouldn't we refer to https://github.com/globus/gsi-openssh/wiki/Internals

Reply:
Sure. I didn't knew this document was moved to the wiki of GSI-OpenSSH on GitHub.

Reply:
Do we allow "rewriting history"? I.e. should I squash the change of the URL in the original commit (not sure if this code comment will then cease to exist) or should I create a new one for this change?

Reply:
I think if you just commit to your 'corrections' branch, the pull request will get updated automatically.

Reply:
Sure, but we'll have another commit cluttering the history then. If we don't care that'd be the easiest way I think.

Reply:
Either way is fine with me. Feel free to squash them in your branch (even if this conversation disappears as a result) or leave it as two commits.
We shouldn't squash in the actual gct-docs repo though.

Reply:
I'm OK with that.